openedx/core/lib/extract_zip.py

```diff
@@ -0,0 +1,57 @@
+"""
+Safe version of zipfile.extractall which does not extract any files that would
+be, or symlink to a file that is, outside of the directory extracted in.
+
+Adapted from:
+http://stackoverflow.com/questions/10060069/safely-extract-zip-or-tar-using-python
+"""
```

Comment:
thought: 💭 This file could be merged with openedx/core/lib/extract_tar.py

Tried to keep the changes minimal by create a new file